src/modules/project/pages1/projectView/views/rewards/hooks/useProjectRewardForm.ts

```diff
@@ -156,7 +156,7 @@ export const useProjectRewardForm = ({
       description: formData.description,
       shortDescription: formData.shortDescription,
       cost: formData.cost,
-      maxClaimable: formData.maxClaimable,
+      maxClaimable: Number(formData.maxClaimable),
```

Comment:
If it's an empty string, it should be set to null otherwise it will set it to 0.

Reply:
Good catch

Reply:
Good catch